pkg/services/star/star_model.go

```diff
@@ -0,0 +1,36 @@
+package star
+
+import "errors"
+
+var ErrCommandValidationFailed = errors.New("command missing required fields")
+
+type Star struct {
+	Id          int64
```

Comment:
Could this be the right time to name things "ID" vs "Id"? Not sure if such holy wars already happened or not yet. Maybe @sakjur knows?

Reply:
I don’t think anyone would be against that. I have started on ID and URL in my work on snapshots 🙂

Reply:
Good thing that once we adopt UIDs - noone would be able to spell it as `UId`. Well, I hope.

Reply:
ofc not, they're clearly `uiD`